test/myFTClient.spec.js

```diff
@@ -14,7 +14,7 @@ const baseUrl = config.MYFT_API_URL;
 const extraParams = `?noEvent=${config.MYFT_NO_EVENT}&waitForPurge=${config.MYFT_WAIT_FOR_PURGE_ADD}`;
 
 const myftConst = config.myftClientConstants;
-const suppressLogs = false; //for local test if you want logs when test are run
+const suppressLogs = true; //for local test if you want logs when test are run
```

Comment:
Just curiously, could we just put `CONSOLE_LOG_LEVEL=none` before the test command instead of doing it in the spec files?
eg.
```make
test:
    CONSOLE_LOG_LEVEL=none mocha test/**/*.js
```

Reply:
@simonplend  suppressLogs kind of works the otherway round. The logs are stubbed in the client-proxies specs so I added that in as a toggle so we could 'unstub' them if you wanted to see what the tests were logging. I've done this on the specs where I've removed the 'mockAPI' flag to simplify and move away from situations where unit test are not being stubbed, mocked or spied on.

Tried ripping out the stubs and doing 
```
unit-test:
	export KAT_FT_TOOL_ADMIN_ID=00000000-0000-0000-0000-012345678910;
	export CONSOLE_LOG_LEVEL=none;
	mocha 'test/**/*.js' --inline-diffs  --timeout 10000
```
Did not work. Any idea 🤷‍♂️

Reply:
Sorry again @simonplend I'll take this one

Reply:
Nah, oh well it's not that bad

Reply:
BTW sorry @simonplend must stop asking you to PR stuff 😬

Reply:
:joy_cat: